test/elixir_console_web/features/console_test.exs

```diff
@@ -19,4 +19,22 @@ defmodule ElixirConsoleWeb.ConsoleTest do
       |> assert
     end)
   end
+
+  feature "visitor can get suggestions while typing", %{session: session} do
```

Comment:
We could make this test a bit more specific by describing it as "visitor can get suggestions of modules while typing", and then add another test where you type "Enum.c" and you also get suggestions of functions as it could be seen as two separate features. Wdyt?

Reply:
I think that would make sense in case there's anything specific in the UI side regarding modules and functions suggestion with regards to functionality.

If there's not anything special on the UI side about that maybe just 1 test case for the suggest feature suffices and we could have that specific coverage on a unit test of the Sandbox, or the actual place where that difference matters?